package.json

```diff
@@ -26,6 +26,19 @@
     "main": "dist/main.js",
     "module": "dist/module.js",
     "types": "dist/module.d.ts",
+    "type": "module",
+    "exports": {
+        ".": {
+            "types": "./dist/module.d.ts",
+            "import": "./dist/es.js",
+            "require": "./dist/module.js"
+        },
+        "./react": {
+            "types": "./react/dist/types/index.d.ts",
+            "import": "./react/dist/esm/index.js",
+            "require": "./react/dist/umd/index.js"
+        }
+    },
```

Comment:
i've been sort of ignoring the module/bundling changes in JS since they seem pretty volatile and i was hoping for things to settle and i could learn it once 🤣 

can you explain what this does/means?

i'm mostly wondering if we'll have unexpected consequences in other environments than you're testing for, i've found sometimes apparently benign changes throw up weird errors for folk

(my guess is this is safe but 🙈)